app/views/notifications/_comment.html.erb

```diff
@@ -0,0 +1,10 @@
+<% commentable = comment.commentable %>
```

Comment:
@aapomm I just noticed: we must handle the case when the comments has been deleted ?

Reply:
We create an Activity on comment creation, update and deletion... but I don't see why we need a *notification* for update and deletion.

I can't think of any service I use that sends me a notification when a post/comment/whatever is *deleted or edited* as opposed to just published/created.

If I'm right this partial can be much much simpler because e.g. we don't need to bother with `presenter.verb` as we already know what the verb will be ("created"!)

Reply:
What's the outcome of this conversation? I think @georgemillo has a point re: Notifications only needed on create?

Reply:
Looks like aaron [removed](https://github.com/dradis/dradis-ce/commit/db7dcb2d7533e6058e4b844a83498fbddf82d86e#diff-ecab1e17a4becfd34c67ccabc4181a83L36) the `verb` method from the presenter so my comment is outdated? Although we may come back to this when we start adding new types of notification.

C/f [my comment here](https://github.com/dradis/dradis-ce/pull/303/files#r203689765) which is relevant.